### ISSUE TITLE
`err="parsing frame 0: reading channel_id: unexpected EOF"` on celestia-develop-apr-24-392b8872 branch

### ISSUE THREAD

**Issue by jcstein** [celestiaorg/optimism#19]
OP-node is not able to parse the Frames that contain the data from Celestia

[Here are the relevant logs](https://gist.github.com/jcstein/15eaa67218ae4fe0bd064e40b5e8922f)

**Comment by jcstein** [comment]
Still getting this error when running off of #20 

https://gist.github.com/jcstein/3d6b812b249123134046e8efc96d2460

**Comment by jcstein** [comment]
[Here are the updated instances of "parsing frame" in the Docker compose logs](https://gist.github.com/jcstein/93cdb9db616e8ec329258b61ef02fce9) after 21301 blocks on the L2. Occurs 2092 times

**Comment by jcstein** [comment]
Closed in #22 

I can confirm that I have tested this on #22, and see no instances of frame parsing error:

```bash
root@jcs-optimism:~/optimism/ops-bedrock# docker-compose logs | grep "parsing frame"
root@jcs-optimism:~/optimism/ops-bedrock# docker-compose logs | grep "Failed to parse frames"
root@jcs-optimism:~/optimism/ops-bedrock#
```

**Comment by jcstein** [comment]
experiencing this on https://github.com/celestiaorg/optimism/pull/56

```bash
op-node_1      | t=2023-05-23T20:57:30+0000 lvl=warn msg="unable to retrieve data from da"        origin=0xca02eb0de3e674987e314679fc9e655aae82597e36650f283402313e66dc27ba:18 err="share sequence has 1 shares but needed 0 shares"
op-node_1      | t=2023-05-23T20:57:30+0000 lvl=warn msg="Failed to parse frames"                 origin=0xca02eb0de3e674987e314679fc9e655aae82597e36650f283402313e66dc27ba:18 err="parsing frame 0: reading channel_id: unexpected EOF"
```
